src/OrbitQt/orbittreeview.cpp

```diff
@@ -174,7 +174,7 @@ void OrbitTreeView::Refresh(RefreshMode refresh_mode) {
 
 void OrbitTreeView::resizeEvent(QResizeEvent* event) {
   const bool width_resized = event->size().width() != event->oldSize().width();
-  if (width_resized && model_ != nullptr && model_->GetDataView()) {
+  if (width_resized && model_ != nullptr && (model_->GetDataView() != nullptr)) {
```

Comment:
```suggestion
  if (width_resized && model_ != nullptr && model_->GetDataView() != nullptr) {
```
Just for consistency with the rest of the condition.

Reply:
Done.